src/Components/WriteOnlyButton/WriteOnlyButton.js

```diff
@@ -13,6 +13,11 @@ const WriteOnlyButton = (props) => {
   return (
     <Button isDisabled={isDisabled} {...buttonProps}>
       {children}
+      {isDisabled && (
+        <Tooltip content={<div>Your tooltip content goes here</div>}>
+          <span></span>
+        </Tooltip>
+      )}
```

Comment:
We shouldn't add this tooltip - especially if it just says "Your tooltip content goes here"